lib/changelog.js

```diff
@@ -0,0 +1,237 @@
+var Async = require('async');
+var Config = require('getconfig');
+var Utils = require('./utils');
+var Querystring = require('querystring');
+
+var internals = {};
+
+internals.dependencyRegex = /(?:update|upgrade) (\w+)\/(\w+) to (.*) from (.*)/i;
+
+internals.hapiOrg = 'hapijs';
+
+internals.perPage = 100;
+
```

Comment:
two spaces between top level items. but why not just do

internals = {
  dependencyRegex: ...
  hapiOrg: 'hapijs',
  perPage: 100
};